### ISSUE TITLE
ModelExtras Bugs Mega Thread

### ISSUE THREAD

**Issue by CanerKaraca23** [user-grinch/ModelExtras#47]
**All issues on ModelExtras that i encountered, check videos on below.**

Vehicles used on videos: 
[Vehicles.zip](https://github.com/user-attachments/files/18624288/Vehicles.zip)

My mod list:
[Mods.txt](https://github.com/user-attachments/files/18238008/Mods.txt)

1. Coronas are not looking pretty compared to AVS and ImVehFt. AVS have more intense and sharp coronas and it's better.
2. Some coronas are can be seen from angles they shouldn't be seen.
3. All coronas are flickering. (This issue is not available on AVS and ImVehFt)

### Videos:

1. issue:

How it's on AVS:

https://github.com/user-attachments/assets/c343de23-ffed-4c08-a4ca-419b8b84bfee

https://github.com/user-attachments/assets/a4e0fdd4-1b46-4266-85f4-eb0a7a4cf6c2

How it's on ModelExtras:

https://github.com/user-attachments/assets/be5bbabd-ebc0-44ef-a664-e502fd920bbb

https://github.com/user-attachments/assets/6b179786-7631-4f63-9ff8-ac59b1f223ff

2. issue:

How it's on AVS:

![Image](https://github.com/user-attachments/assets/b6b5f0ea-aa8a-4668-a70f-a0a7a6b3d242)

How it's on ModelExtras:

![Image](https://github.com/user-attachments/assets/b6b91826-5e75-462b-aac2-dbf444bfdb5e)

3. issue:

https://github.com/user-attachments/assets/23cfbc03-b466-4052-8a1c-466172e4de10

**Comment by CanerKaraca23** [comment]
@user-grinch 

''Vehicles with opened sirens headlight becomes crazy when you hit somewhere.'' this issue still exists on latest build.

**Comment by user-grinch** [comment]
Is the enforcer model you send above correct? Doesn't seem to match the one in the video.

**Comment by CanerKaraca23** [comment]
> Is the enforcer model you send above correct? Doesn't seem to match the one in the video.

Yeah, it's not that, i can't find the car on the video on my folders, i don't know what happened and why the car i uploaded is white.

**Comment by user-grinch** [comment]
Note: The eml -> json files need to be converted again for this fix to work.

**Comment by CanerKaraca23** [comment]
I got crash on startup after new build @user-grinch 

[gta_sa.exe_2025-01-31_14-36-59.log](https://github.com/user-attachments/files/18617690/gta_sa.exe_2025-01-31_14-36-59.log)

**Comment by CanerKaraca23** [comment]
@user-grinch Blue texture not blinking

https://discord.com/channels/689515979847237649/1129105388168556554/1334975444855947357

**Comment by CanerKaraca23** [comment]
@user-grinch There is something wrong with headlight blinking

https://github.com/user-attachments/assets/3a05b5f4-317b-492f-9610-b01da5d804d8

**Comment by CanerKaraca23** [comment]
@user-grinch Updated the thread, the car on videos was not enforcer, it was fbi truck, uploaded the models.

**Comment by user-grinch** [comment]
I'm going to keep adding stuff as I go here,

```Log says there is a sound mod for m4 and minigun shoot sound effect but i don't have and in game, m4 and minigun shoot sound effect not working.```
I mistakenly shipped my files with the ModelExtras download. As far as the issue with audio, I don't know why it's happening atm. No one else reported such a thing yet. Might need to debug more. 
It's better to keep this a separate this from the reset.

**Comment by CanerKaraca23** [comment]
`Some coronas are can be seen from angles they shouldn't be seen.`
Yes it still happens.

`Vehicles with opened sirens headlight becomes crazy when you hit somewhere on every vehicle that have sirens.`
Yes it still happens.

`All coronas are flickering.`
Yes it still happens.

`Roundable things are not rounds.`
It's on police cars. In RVP's ReSmooth version, KaiQ made police cars that have roundable things on siren place. They was rounding before but idk why they not now.

`Left headlight texture not blinking and extra headlight textures not blinking on all vehicles.`
It happens on all my vehicles, just pick the ambulance on the thread.

`Indicator lights shadows positions are wrong on motorcycles.`
It's from RVP ReSmooth, [faggio.zip](https://github.com/user-attachments/files/18630311/faggio.zip)

@user-grinch 

Also there is a new bug unfortunately. It appeared on last build:

https://github.com/user-attachments/assets/c565c617-9a13-46fb-b64c-eaa492a8b795

**Comment by user-grinch** [comment]
Check faggio. It's a model issue. The author used front light dummy for the rear

![Image](https://github.com/user-attachments/assets/6e74d071-4575-45a0-9372-a45c135ad13c)

**Comment by CanerKaraca23** [comment]
@user-grinch Looks like all the issues left are fixed! Coronas are looking more pretty and not flickering anymore! Also angles are fine too!
